src/Command/PullRequest/PullRequestMergeCommand.php

```diff
@@ -312,7 +302,7 @@ private function getCommitsString(array $commits)
         return $commitsString;
     }
 
-    private function getPrType(InputInterface $input, OutputInterface $output, $prType)
```

Comment:
so the stylehelper can house the interactivity of the input and it house the output as well, hmm interesting